src/plugin.js

```diff
@@ -17,7 +17,16 @@ import mix from '../utils/mix.js';
  */
 export default class Plugin {
 	/**
-	 * Creates a new Plugin instance.
+	 * Creates a new Plugin instance. This is first step of a plugin initialization.
+	 * See also {@link #init} and {@link #afterInit}.
+	 *
+	 * A plugin is always instantiated after its {@link #requires dependencies} and the
+	 * {@link #init} and {@link #afterInit} methods are called in the same order.
+	 *
+	 * Usually, you'll want to put your plugin's initialization code in the {@link #init} method.
```

Comment:
The rule of thumb is that professional documentation never speaks directly to the reader. Instead:

> Usually, plugin's initialization code lands in the {@link #init} method.